examples/get_first_n_users.py

```diff
@@ -0,0 +1,26 @@
+__author__ = 'JuriaSan'
+
+import requests
+
+
+max_users_num = 50
+count = 0
+page_num = 0
+has_next = True
+users = []
+stop = False
+while not stop and has_next:
+    page_num += 1
+    url = 'https://stepic.org/api/users'.format(page_num)
+    request = requests.get(url).json()
+    users = request['users']
+    has_next = request['meta']['has_next']
+    for user in users:
+        if len(users) >= max_users_num:
+            stop = True
+            break
+        else:
+            users.append(user)

```

Comment:
Why is it "bad" and do you have any explanation why it is bad or incorrect? Then, what is correct and what steps should I perform instead of appending to the list (which seems to be obvious solution) if I want to save the users?